drivers/hmis/app/graphql/types/hmis_schema/ce_referral.rb

```diff
@@ -9,23 +9,30 @@
 module Types
   class HmisSchema::CeReferral < Types::BaseObject
     # object is a Hmis::Ce::Referral
+
```

Comment:
we should probably add object-level auth but I couldn't figure out how to do it performantly yet - will create follow-up